chia/consensus/constants.py

```diff
@@ -80,15 +80,15 @@ class ConsensusConstants:
     # number of consecutive plot ids required to be distinct
     UNIQUE_PLOTS_WINDOW: uint8
 
-    def replace(self, **changes: object) -> "ConsensusConstants":
+    def replace(self, **changes: Any) -> "ConsensusConstants":
         return dataclasses.replace(self, **changes)
 
     def replace_str_to_bytes(self, **changes: Any) -> "ConsensusConstants":
         """
         Overrides str (hex) values with bytes.
         """
 
-        filtered_changes = {}
+        filtered_changes: Any = {}
```

Comment:
On my phone now but I'll explore this later today.  Not sure why it needs to be quite this unknown what this is.

Reply:
without this, the call to `replace()` below fails with a type error

Reply:
```
chia/consensus/constants.py:103: error: Argument 2 to "replace" of "ConsensusConstants" has incompatible type "**dict[str, bytes]"; expected "uint32"  [arg-type]
chia/consensus/constants.py:103: error: Argument 2 to "replace" of "ConsensusConstants" has incompatible type "**dict[str, bytes]"; expected "uint8"  [arg-type]
chia/consensus/constants.py:103: error: Argument 2 to "replace" of "ConsensusConstants" has incompatible type "**dict[str, bytes]"; expected "uint64"  [arg-type]
chia/consensus/constants.py:103: error: Argument 2 to "replace" of "ConsensusConstants" has incompatible type "**dict[str, bytes]"; expected "uint128"  [arg-type]
chia/consensus/constants.py:103: error: Argument 2 to "replace" of "ConsensusConstants" has incompatible type "**dict[str, bytes]"; expected "int"  [arg-type]
chia/consensus/constants.py:103: error: Argument 2 to "replace" of "ConsensusConstants" has incompatible type "**dict[str, bytes]"; expected "bytes32"  [arg-type]
```

Reply:
I took this change (which also required adding `Dict` to the import statement)